tests/helpers/compute_helpers.py

```diff
@@ -72,6 +71,29 @@ def build_and_send_ddo_with_compute_service(
             custom_services_args=[],
             timeout=timeout,
         )
+    elif asset_type == "stored_in_arweave":
+        arweave_file_object = {
+            "type": "arweave",
+            "transactionId": "cZ6j5PmPVXCq5Az6YGcGqzffYjx2JnsnlSajaHNr20w",
+        }
+
+        dataset_ddo_w_compute_service = get_registered_asset(
+            publisher_wallet,
+            unencrypted_files_list=[arweave_file_object],
+            custom_services="vanilla_compute",
+            custom_services_args=[
```

Comment:
this is completely copied. Let's extract a function or assign `standard_compute_args`

Reply:
Done.  I adjusted assets stored in arweave to accept all algorithms (thus eliminating this duplicate code block)